tests/conftest.py

```diff
@@ -337,7 +338,17 @@ def create_pod(username, server_name, payload):
                         }
                     },
                 }
-            ]
+            ],
+            "volumes": [
+                {
+                    "name": server_name + "-git-repo",
```

Comment:
should this be using the pv name?

Reply:
I would need to mock up the list of pvcs in k8s as well as adding/deletion when a session is started in order to make the units tests work with the PVCs. This was a lot easier. If you look at the top of the `conftest.py` file you will see that I specifically set the environment variable flag that indicates that PVCs are enabled to false. I am not sure it is worth pursuing this further because the tests change a lot with amalthea and a lot of these things that require extensive mocking move into the integration tests that use kind and an actual deployment.